esmvaltool/diag_scripts/perfmetrics/portrait_plot.py

```diff
@@ -0,0 +1,476 @@
+"""Overview plot for performance metrics.
+
+Description
+-----------
+This diagnostic provides plot functionalities for performance metrics.
+The multi model overview heatmap might be useful for different
+tasks and therefore this diagnostic tries to be as flexible as possible.
+X and Y axis, grouping parameter and slits for each rectangle can be
+configured in the recipe. All *_by parameters can be set to any metadata
+key. To split by 'reference' this key needs to be set as extra_facet in recipe.
+
+Author
+------
+Lukas Ruhe (Universität Bremen, Germany)
+Diego Cammarano
+
+Configuration parameters through recipe:
+----------------------------------------
+normalize: str or None, optional
+    ('mean', 'median', 'centered_mean', 'centered_median', None).
+    Subtract median/mean if centered. Divide by median/mean if not None.
+    By default None.
```

Comment:
```suggestion
    By default 'centered_median'.
```

Just came across this, no full review yet 😁